### PR TITLE
Issue 218

### PR DESCRIPTION
**Pull request by rmroot**
connects #218 

hides export buttons and removes unused export logic for .csv files no longer supported

(has icon PR changes)

### CODE REVIEW

File: src/app/shared/helper-services/export-to-template.service.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http'
+import * as XLSX from 'xlsx';
+import { UtilityMeterdbService } from 'src/app/indexedDB/utilityMeter-db.service';
+import { IdbUtilityMeter } from 'src/app/models/idb';
+@Injectable({
```
Comment (nbintertech) [inline src/app/shared/helper-services/export-to-template.service.ts:6]: Don't provide in root scope, provide directly in using module

File: src/app/shared/helper-services/export-to-template.service.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+  exportToTemplate() {
+    this.http.get("assets/csv_templates/VERIFI-Import-Data.xlsx", { responseType: "arraybuffer" }).subscribe(
+      data => {
+        console.log(data);
+        let blob = new Blob([data], { type: 'application/mx-excel' });
+        this.readWorkbook(blob);
+      },
+      error => {
+        console.log("ERROROROOROR")
```
Comment (nbintertech) [inline src/app/shared/helper-services/export-to-template.service.ts:21]: Still need console logging? multiple in this service

**Review by nbintertech — CHANGES_REQUESTED** [review]

File: src/app/shared/helper-services/export-to-template.service.ts (hunk 1 above)
Comment (rmroot) [reply src/app/shared/helper-services/export-to-template.service.ts:6]: We are using root scope in this project. Haven't broken up code into modules

File: src/app/shared/helper-services/export-to-template.service.ts (hunk 2 above)
Comment (rmroot) [reply src/app/shared/helper-services/export-to-template.service.ts]: This service was unused. deleted

**Review by nbintertech — APPROVED** [review]

**Pull request merged**